### ISSUE TITLE
FUSE update to 4.0.2 from Preferences Pane is (somewhat) broken

### ISSUE THREAD

**Issue by mcicak** [osxfuse/osxfuse#744]
1. Open System Preferences
2. Open FUSE
3. Enable Beta versions
4. Click Update
5. Update fails with error message
6. Tap on FUSE in System Preferences; error shows: “FUSE” can’t be opened because it isn’t available.
7. Right click on FUSE icon and 'Remove FUSE Preferences Pane"; error "The operation couldn’t be completed. (install assistant error -200.)" is displayed

8. After closign System Preferences and opening again, macFUSE is displayed in PreferencesPane reporting version 4.0.2 is installed.

Expected:
OSXFUSE 3.11.2 should be updated to macFUSE 4.0.2

Actual:
Error occured during update and afterwards osxfuse is in broken state.

Environment:
macOS Catalina 10.15.7 (19H2)
MacBook Pro (15-inch, 2019)

**Comment by bfleischer** [comment]
Have you tried closing (Quit) System preferences and then opening it again?

**Comment by bfleischer** [comment]
> Update fails with error message

What exactly is the error message?

**Comment by mcicak** [comment]
After reopening system preferences, state was correct: macFUSE was displayed instead of previous FUSE.
However the initial experience of updating osxfuse to macFUSE beta was broken.
I'm not currently at the mbp but I stated the errors I got in the original message in this issue. They were shown in alert dialogs.

**Comment by bfleischer** [comment]
Your ticket description does not contain the error message for step 5, or I am missing something. In any case, it would be great if you could update the description or point me to the message. Thanks.

**Comment by mcicak** [comment]
Mhm I see. Ok, I'll investigate and try to reproduce and get back when I have more info.

**Comment by mcicak** [comment]
Here are some more details: I tried to update FUSE 3.11.2 on another MBP to latest beta version.

When I pressed Update FUSE button it started the updated process which ended in showing System Preferences view with 'There was an error in FUSE preferences.' alert. Tapping the FUSE icon results in 'FUSE can't be opened because it isn't available.'.

Not sure if this is perhaps a bug in macOS's System Preferences dialog?

<img width="780" alt="Screenshot 2020-11-16 at 16 48 32" src="https://user-images.githubusercontent.com/1625174/99275647-0d323b80-282c-11eb-8168-b87c4f3ad84d.png">

<img width="780" alt="Screenshot 2020-11-16 at 16 48 44" src="https://user-images.githubusercontent.com/1625174/99275659-102d2c00-282c-11eb-9402-46e0fdcbcc1b.png">

<img width="780" alt="Screenshot 2020-11-16 at 16 48 52" src="https://user-images.githubusercontent.com/1625174/99275667-128f8600-282c-11eb-9093-21b2b869eff0.png">

**Comment by scrutinizer11** [comment]
I have an issue that is very similar but in my case, I ended up having two FUSE panes - '"FUSE" and "macFUSE". The OS is macOS 10.9. It showed the update was available, so I wanted to update from 3.11.2 but it stuck on "Updating". I then closed Preferences and updated manually by downloading the package from the official site. It reported installation success however I got 2 FUSE icons. Acting on intuition I went to _/Library_ and deleted the old entry from the _Filesystems_ subfolder. I also found a folder inside _/Library/Caches_ bearing the name of the old OSFUSE version with restricted access permissions, with the prohibition sign stamped on it.  I deleted that one as well, emptied Trash and restarted but I still see the old icon pertaining to the old version of FUSE, only this time it's not FUSE's native icon but that of System Preferences itself. I managed to collect some log entries in Console which is what I believe may point to the cause but I'm not a programmer and have no clue about what that means provided that's even relevant. I post the screenshots and a Console log with the entries in question.
In case you wonder I got FUSE because I run the application SyncMate that requires this component.
![Screen Shot 2021-03-05 at 00 23 16](https://user-images.githubusercontent.com/45983734/110040589-bce3d880-7d4b-11eb-98cb-41b8b8eebd66.png)
![Screen Shot 2021-03-05 at 00 23 35](https://user-images.githubusercontent.com/45983734/110040597-be150580-7d4b-11eb-8c07-bd9e551545ca.png)


[osxfuse.log](https://github.com/osxfuse/osxfuse/files/6087035/osxfuse.log)
